### DISCUSSION TITLE
docker安装后，PotPlayer 64 bit无法播放，提示”服务器已关闭或地址错误“

### DISCUSSION THREAD

**Discussion by tommyhe1217** [Guovin/iptv-api#758] [Q&A] [answered]
docker安装后，PotPlayer 64 bit无法播放，提示”服务器已关闭或地址错误“，接口内容和测速都没问题，地址栏输入ip+端口/m3u，可以下载m3u文件，导入文件播放也提示”服务器已关闭或地址错误“

**Comment by Guovin** [comment, marked as answer]
修改配置open_url_info = False

**Comment by tommyhe1217** [comment]
services:
  iptv-api:
    image: guovern/iptv-api
    container_name: iptv-api
    environment:
        open_update_info: False
    ports:
      - "8755:8000"
    volumes:
      - ./volume2/docker/iptv-api/config:/iptv-api/config
      - ./volume2/docker/iptv-api/output:/iptv-api/output
    restart: always

配置如上，相关问题还是没有得到解决，显示同样的错误。

Govin ***@***.***> 于2024年12月30日周一 10:10写道：

**Reply by Guovin** [reply]
所有配置修改需要挂载config，修改config/config.ini文件后重新运行

**Comment by tommyhe1217** [comment]
修改config/config.ini文件，在文件最后增加一行下面的内容
open_update_info = False

PotPlayer 64 bit无法播放，提示“服务器已关闭或地址错误”
不过TV-BOX可以播放，也就可以了，谢谢大佬的解答。

Govin ***@***.***> 于2024年12月30日周一 10:27写道：

**Reply by Guovin** [reply]
配置文件本来就含有open_url_info参数，直接修改就好，不要追加重复的配置，否则无法生效

**Comment by tommyhe1217** [comment]
我的配置文件没有 open_update_info参数，
和 open_update相关的就下面2条，所以我才添加的，特意确认了的

open_update = True
open_update_time = True

Govin ***@***.***> 于2024年12月30日周一 11:19写道：

**Reply by Guovin** [reply]
抱歉说错了，是open_url_info

**Comment by tommyhe1217** [comment]
哈哈，我自己也猜想会不会是你说错了，原来真的如此，谢谢大佬。

Govin ***@***.***> 于2024年12月30日周一 13:34写道：
